src/highdicom/sr/templates.py

```diff
@@ -3516,23 +3681,24 @@ def __init__(
             relationship_type=RelationshipTypeValues.HAS_ACQ_CONTEXT
         )
         self.append(modality_item)
-        frame_of_reference_uid_item = UIDRefContentItem(
-            name=CodedConcept(
-                value='112227',
-                meaning='Frame of Reference UID',
-                scheme_designator='DCM'
-            ),
-            value=frame_of_reference_uid,
-            relationship_type=RelationshipTypeValues.HAS_ACQ_CONTEXT
-        )
-        self.append(frame_of_reference_uid_item)
+        if 'FrameOfReferenceUID' in image:
+            frame_of_reference_uid_item = UIDRefContentItem(
+                name=CodedConcept(
+                    value='112227',
+                    meaning='Frame of Reference UID',
+                    scheme_designator='DCM'
+                ),
```

Comment:
For many of these, wouldn't it be preferable to use the codes in `pydicom`? E.g.

```suggestion
                name=codes.FrameOfReferenceUID,
```

Reply:
Agreed. I've changed these on my PR's code - should I change this in the rest of templates.py or make that a separate PR?